tmc-prototype/CentralNode/CentralNode/CONST.py

```diff
@@ -0,0 +1,61 @@
+# In/Out command constants
+CMD_SET_STOW_MODE = "SetStowMode"
+CMD_SET_STANDBY_MODE = "SetStandbyLPMode"
+CMD_SET_OPERATE_MODE = "SetOperateMode"
+
+
+
+#Event messages
+EVT_SA1_NODE = "tm_subarray_node/1"
+EVT_SA2_NODE = "tm_subarray_node/2"
```

Comment:
Updated subarrayHealthStateCallback to avoid usage of the constant variable for each SubarrayNode. Commit id is 2c18d86d7f13f585e192184d59f1619ac996930a.